platform/broadcom/docker-syncd-brcm-rpc.dep

```diff
@@ -0,0 +1,10 @@
+
+DPATH       := $($(DOCKER_SYNCD_BRCM_RPC)_PATH)
+DEP_FILES   := $(SONIC_COMMON_FILES_LIST) rules/docker-iccpd.mk rules/docker-iccpd.dep   
```

Comment:
Why not add platform/broadcom/docker-syncd-brcm-rpc.mk platform/broadcom/docker-syncd-brcm-rpc.dep?
Seems that the two files rules/docker-iccpd.mk rules/docker-iccpd.dep do not exist.

Can we add the $module.mk and $module.dep automatically, not need to add in each of the dep file?

Reply:
Agreed, We will address this in next phase.